requirements.txt

```diff
@@ -8,4 +8,5 @@ requests-mock==1.11.0
 coverage==7.3.2
 scikit-learn~=1.3.2
 boto3==1.34.30
+moto==4.0.0
```

Comment:
should we use a separate requirements.txt for the authorizer, to avoid mixing cicd dependencies with app dependencies?